### PR TITLE
[NEW]  Swipe to reply message

### PR DESCRIPTION
**Pull request by Adizbek**
@RocketChat/android

#### Screenshots or GIF for the change:
https://youtu.be/lD3xU6pBWEk

### CODE REVIEW

File: app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt [hunk 1]
```diff
@@ -28,17 +28,13 @@ import androidx.core.view.isVisible
...
 import chat.rocket.android.R
 import chat.rocket.android.analytics.AnalyticsManager
 import chat.rocket.android.analytics.event.ScreenViewEvent
-import chat.rocket.android.chatroom.adapter.ChatRoomAdapter
-import chat.rocket.android.chatroom.adapter.CommandSuggestionsAdapter
-import chat.rocket.android.chatroom.adapter.EmojiSuggestionsAdapter
-import chat.rocket.android.chatroom.adapter.PEOPLE
-import chat.rocket.android.chatroom.adapter.PeopleSuggestionsAdapter
-import chat.rocket.android.chatroom.adapter.RoomSuggestionsAdapter
+import chat.rocket.android.chatroom.adapter.*
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt]: Avoid using wildcard import.

File: app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt [hunk 2]
```diff
@@ -813,6 +809,33 @@ class ChatRoomFragment : Fragment(), ChatRoomView, EmojiKeyboardListener, EmojiR
...
         recycler_view.addOnScrollListener(fabScrollListener)
+
+        val touchCallback: ItemTouchHelper.SimpleCallback = object : ItemTouchHelper.SimpleCallback(0, ItemTouchHelper.LEFT) {
+            override fun onMove(recyclerView: RecyclerView, viewHolder: RecyclerView.ViewHolder, target: RecyclerView.ViewHolder): Boolean {
+                return true
+            }
+
+            override fun onSwiped(viewHolder: RecyclerView.ViewHolder, direction: Int) {
+                if (viewHolder is MessageViewHolder) {
+                    citeMessage(chatRoomName, chatRoomType, viewHolder.data?.messageId!!, true)
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt]: Avoid using `!!`

**Review by philipbrito — CHANGES_REQUESTED** [review]

File: app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt (hunk 1 above)
Comment (Adizbek) [reply app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt]: Thanks, I'll fix them today.

File: app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt (hunk 2 above)
Comment (Adizbek) [reply app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt]: Fixed

**Comment by Adizbek** [comment]
@filipedelimabrito Can you review?

**Review by philipbrito — APPROVED** [review]
Thanks for sending this PR @Adizbek!
💯

**Pull request merged**